src/components/Image/index.astro

```diff
@@ -0,0 +1,14 @@
+---
+import { Image } from "astro:assets";
+const { src, alt, width, height } = Astro.props;
```

Comment:
are the width and height considered optional? Astro's `Img` class should be able to read height and width of the image asset at build time so i don't think we should have to pass these in. it looks like you're using it like that, just want to be sure.

Reply:
Yep I added some props to be more explicit